### ISSUE TITLE
Configuration issues

### ISSUE THREAD

**Issue by SicorDev** [Gregor-Agnes/newsletter_subscribe#46]
Hi guys!

Great extension, albeit I'd have a few suggestions to improve configuration for TYPO3 10 and onwards.

I spent quite some time to figure out some typoscript settings:

1.
config.language = de
config.locale_all = de_DE.utf8

=> Instead of relying on otherwise superfluous config.language entries, try auto detecting the info from the request. Code might work like that:

```
$requLanguage = $GLOBALS['TYPO3_REQUEST']->getAttribute('language');
$langid = $requLanguage ? substr($requLanguage->getHrefLang(), 0, 2) : "de";
```

2.
plugin.tx_newslettersubscribe.settings.subscribePageUid = 1397
plugin.tx_newslettersubscribe.settings.mailTemplateRootPath = EXT:newsletter_subscribe/Resources/Private/Templates/Mail/

=> Both are required for "unsubscribe", but can neither be configured via constants nor are they present in the plugin. Actually I don't see any constants at all in my installation, which feels like an error? Not sure...

Anyway, carry on with the good work and Happy Holidays!

**Comment by Gregor-Agnes** [comment]
Thx, I will look into it in the next release!

**Comment by Gregor-Agnes** [comment]
1. This is fixed in 02024f05cc543fcb26ad9a3975f62f303211d83d
2. Those are used in the templates and defined in [setup.typoscript](https://github.com/Gregor-Agnes/newsletter_subscribe/blob/cd3404d4f5396ae89a8798cd59f633675d1c8f08/Configuration/TypoScript/setup.typoscript)
